cpr/response.cpp

```diff
@@ -13,8 +13,16 @@ Response::Response(std::shared_ptr<CurlHolder> curl, std::string&& p_text,
     char* url_string{nullptr};
     curl_easy_getinfo(curl_->handle, CURLINFO_EFFECTIVE_URL, &url_string);
     url = Url(url_string);
+#if LUBCURL_VERSION_NUM < 0x075500
```

Comment:
```suggestion
#if LUBCURL_VERSION_NUM < 0x070401
```
Based on the docs for `CURLINFO_SIZE_DOWNLOAD` and `CURLINFO_SIZE_UPLOAD`, shouldn't this be `0x070401`, since it's available since `7.4.1`.

https://curl.haxx.se/libcurl/c/CURLINFO_SIZE_DOWNLOAD.html

Reply:
`CURLINFO_SIZE_DOWNLOAD_T` is available since 7.55.0 https://curl.haxx.se/libcurl/c/CURLINFO_SIZE_DOWNLOAD_T.html .  I need to convert to hex.

Reply:
I converted to hex; I didn't add a case for < 7.4.1 because it doesn't look like cpr is designed to support that version, and I haven't reviewed other areas of code or tested with the version.